### PR TITLE
Fix: Loans adjustments

### PR DESCRIPTION
**Pull request by lemunozm**
# Description

A couple of minor tasks before the upgrading process

- [x] Update `MaxFeedValues` to `500`
- [x] Borrow proxy type to include `transfer_debt` and token transfer capabilities

### CODE REVIEW

**Comment by mustermeiszer** [comment]
@lemunozm I will take of this in the restricted transfers PR as we also need to add transfer capcabilities to the borrower proxy, if this is fine for you?

**Comment by lemunozm** [comment]
Sure!

File: runtime/altair/src/lib.rs [hunk 1]
```diff
@@ -490,6 +490,7 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 					RuntimeCall::Loans(pallet_loans::Call::propose_write_off_policy{..}) |
 					RuntimeCall::Loans(pallet_loans::Call::apply_write_off_policy{..}) |
 					RuntimeCall::Loans(pallet_loans::Call::update_portfolio_valuation{..}) |
+                    RuntimeCall::Loans(pallet_loans::Call::apply_transfer_debt { .. }) |
```
Comment (mustermeiszer) [inline runtime/altair/src/lib.rs:493]: Missing propose here

**Review by mustermeiszer — COMMENTED** [review]
We are missing a propose in the `NonTransfer` proxz type but otherwise fine.

File: runtime/altair/src/lib.rs (hunk 1 above)
Comment (lemunozm) [reply runtime/altair/src/lib.rs:493]: I was not sure about how to interpret that. Could `propose()` imply a transfer if later somebody apply it? Is `transfer_debt()` taken into account as something that transfers?

Comment (mustermeiszer) [reply runtime/altair/src/lib.rs:493]: By definricion transfer debt does juts transfer internally, right? So it think it is fine. 

If you think differently, then I would remove also the apply one.

Comment (lemunozm) [reply runtime/altair/src/lib.rs:493]: If transferring the debt locally counts as a transfer, then we should not add it
If it's not a transfer, we should add it

Comment (mustermeiszer) [reply runtime/altair/src/lib.rs:493]: He is off. Let's go with adding it!

transferring is just concerned for the acting account to move their funds which will not be the case here.

Comment (lemunozm) [reply runtime/altair/src/lib.rs:493]: Make sense!

**Review by mustermeiszer — APPROVED** [review]

File: runtime/altair/src/lib.rs (hunk 1 above)
Comment (hieronx) [reply runtime/altair/src/lib.rs:494]: Agreed with @mustermeiszer, transferring debt is separate (and much less impactful from security perspective) from transferring stablecoins.

**Pull request merged**